src/Build/BuildCheck/Infrastructure/BuildCheckCentralContext.cs

```diff
@@ -23,7 +23,10 @@ public BuildCheckCentralContext(IConfigurationProvider configurationProvider, Ac
 
     private record CallbackRegistry(
         List<(CheckWrapper, Action<BuildCheckDataContext<EvaluatedPropertiesCheckData>>)> EvaluatedPropertiesActions,
+#pragma warning disable CS0618 // Type or member is obsolete
         List<(CheckWrapper, Action<BuildCheckDataContext<ParsedItemsCheckData>>)> ParsedItemsActions,
```

Comment:
if we operate under Experimental namespace, why we couldn't just remove it?

Reply:
It is possible. Though I want to be a bit more gracefull to prevent impression that it's not a good idea to author extensions yet